.cirrus.yml

```diff
@@ -0,0 +1,79 @@
+# Reference:
+#   - https://cirrus-ci.org/guide/writing-tasks/
+#   - https://cirrus-ci.org/guide/linux/
+#   - https://hub.docker.com/_/gcc/
+#   - https://hub.docker.com/_/python/
+
+#
+# Global defaults.
+#
+container:
+  image: python:latest
+  cpu: 2
+  memory: 4G
+
+
+env:
+  # Maximum cache period (in weeks) before forcing a new cache upload.
+  CACHE_PERIOD: "2"
+  # Increment the build number to force new mambaforge cache upload.
+  MAMBA_CACHE_BUILD: "0"
+  # Increment the build number to force new cf-units cache upload.
+  CFUNITS_CACHE_BUILD: "0"
+  # Base environment conda packages to be installed
+  MAMBA_CACHE_PACKAGES: "pip conda-lock"
+  # Increment the build number to force new pip cache upload.
+  PIP_CACHE_BUILD: "0"
+
+
+#
+# Testing (Linux)
+#
+linux_task:
+  auto_cancellation: true
+  matrix:
+    env:
+      PY_VER: "3.7"
+    env:
+      PY_VER: "3.8"
+    env:
+      PY_VER: "3.9"
+      COVERAGE: "codecov"
+  name: "${CIRRUS_OS}: py${PY_VER}"
+  container:
+    image: gcc:latest
+  env:
+    PATH: ${HOME}/mambaforge/bin:${PATH}
+  mamba_cache:
+    folder: ${HOME}/mambaforge
+    fingerprint_script:
+      - wget --quiet https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh -O mambaforge.sh
```

Comment:
Just realised that this should be part of the `populate_script`, otherwise it's always pulling down mamba, and that's not always necessary

Reply:
Nice catch! 78a85ad79790133088dbbb3bcb06631dd4cb8c32

Reply:
Ahhhh actually we do need it there, as the very next line does a `sha256sum` of it! Sorry, I'm being dense :stuck_out_tongue_closed_eyes:

Reply:
I thought this ought to work - if the fingerprint errors doesn't it default to populating? Then next time it will be here?

Reply:
Ah I see now - this exists outside of the cache recovery, so not downloading the file during fingerprinting means it will _never_ be there. Have now reverted.

51a82523f0efe1ea2af1f9588395c61224a3980f